src/test/java/uk/gov/digital/ho/hocs/client/ComplaintDataTest.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.digital.ho.hocs.client;
+
+import org.junit.Test;
+import uk.gov.digital.ho.hocs.client.casework.dto.UKVIComplaintCorrespondent;
+
+import java.time.LocalDate;
+
+import static org.junit.Assert.assertEquals;
+import static uk.gov.digital.ho.hocs.testutil.TestFileReader.getResourceFileAsString;
+
+public class ComplaintDataTest {
+
+    @Test
+    public void shouldGetDateReceived() {
+        ComplaintData complaintData = new ComplaintData(getResourceFileAsString("staffBehaviour.json"));
+        assertEquals(LocalDate.parse("2020-10-03"), complaintData.getDateReceived());
+    }
+
+    @Test
+    public void shouldGetComplaintType() {
+        ComplaintData complaintData = new ComplaintData(getResourceFileAsString("staffBehaviour.json"));
+        assertEquals("POOR_STAFF_BEHAVIOUR", complaintData.getComplaintType());
+    }
+
+    @Test
+    public void shouldGetUkviComplaintApplicantCorrespondent() {
+        ComplaintData complaintData = new ComplaintData(getResourceFileAsString("applicantCorrespondent.json"));
+        UKVIComplaintCorrespondent ukviComplaintApplicantCorrespondent = complaintData.getUkviComplaintCorrespondent();
+        assertEquals("COMPLAINT", ukviComplaintApplicantCorrespondent.getType());
+        assertEquals("occaecat Lorem", ukviComplaintApplicantCorrespondent.getFullname());
+        assertEquals("sss@uevptde.com", ukviComplaintApplicantCorrespondent.getEmail());
+        assertEquals("01772 700806", ukviComplaintApplicantCorrespondent.getTelephone());
```

Comment:
There are fake numbers provided by OFCOM to avoid accidentally including a real phone number: https://www.ofcom.org.uk/phones-telecoms-and-internet/information-for-industry/numbering/numbers-for-drama

Reply:
Nice, I've learnt something today! Changed...